projects/demo/src/app/datepicker/datepicker.demo.html

```diff
@@ -23,6 +23,18 @@ <h2>Datepicker</h2>
     <li><a [routerLink]="['./reactive-forms']">Reactive Forms</a></li>
     <li><a [routerLink]="['./css-regression']">CSS Regression</a></li>
     <li><a [routerLink]="['./locale-data']">Locale Data</a></li>
+    <li>
+      <a [routerLink]="['./date-range-picker']"
+        >Date Range Picker
+        <span class="badge badge-success"> New </span>
+      </a>
+    </li>
+    <li>
+      <a [routerLink]="['./enhanced-date-picker']"
+        >Enhanced Date Picker
```

Comment:
Maybe we should name it "Custom Date Format" or something in that direction. "Enhanced" is too broad.